libcnb-data/src/buildpack.rs

```diff
@@ -282,6 +288,8 @@ pub enum BuildpackTomlError {
 mod tests {
     use super::*;
 
+    type GenericBuildpackToml = BuildpackToml<Option<toml::value::Table>>;
```

Comment:
I'll update the existing tests in this file to use this new type (and other test cleanup) after this merges.